ahmad/week 2/no3.js

```diff
@@ -0,0 +1,60 @@
+function SeleksiNilai(nilaiAwal, nilaiAkhir, dataArray) {
+  try {
+    if (nilaiAwal === "") {
+      throw "Nilai awal kosong";
+    } else if (isNaN(nilaiAwal)) {
+      throw "Nilai awal harus berupa Angka";
+    } else if (nilaiAkhir === "") {
+      throw "Nilai akhir kosong";
+    } else if (isNaN(nilaiAkhir)) {
+      throw "Nilai akhir harus berupa Angka";
+    } else if (nilaiAkhir < nilaiAwal) {
+      throw "Nilai akhir harus lebih besar dari nilai awal";
+    }
```

Comment:
try to watch this https://www.youtube.com/watch?v=EumXak7TyQ0&t 
and try to implement to your code